include/extractor/extraction_way.hpp

```diff
@@ -46,6 +46,7 @@ struct ExtractionWay
         roundabout = false;
         circular = false;
         is_startpoint = true;
+        ignore_in_grid = false;
```

Comment:
Could this be left-over code? I wanted to check for it but could not find it in the remaining parts of the PR.